# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xchat/gtk-214.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xchat-2.8.6-orig/src/fe-gtk/xtext.h xchat-2.8.6/src/fe-gtk/xtext.h
--- xchat-2.8.6-orig/src/fe-gtk/xtext.h	2009-01-18 13:42:49.000000000 +0000
+++ xchat-2.8.6/src/fe-gtk/xtext.h	2009-01-18 13:50:00.000000000 +0000
@@ -1,6 +1,7 @@
 #ifndef __XTEXT_H__
 #define __XTEXT_H__
 
+#include <gtk/gtktypeutils.h>
 #include <gtk/gtkadjustment.h>
 #ifdef USE_XFT
 #include <X11/Xft/Xft.h>
@@ -270,6 +271,6 @@
 xtext_buffer *gtk_xtext_buffer_new (GtkXText *xtext);
 void gtk_xtext_buffer_free (xtext_buffer *buf);
 void gtk_xtext_buffer_show (GtkXText *xtext, xtext_buffer *buf, int render);
-GtkType gtk_xtext_get_type (void);
+GType gtk_xtext_get_type (void);
 
 #endif
